x-pack/metricbeat/module/statsd/server/data_test.go

```diff
@@ -323,6 +1150,21 @@ func TestData(t *testing.T) {
 	mbtest.WriteEventToDataJSON(t, mbevent, "")
 }
 
+func TestDataAirflow(t *testing.T) {
```

Comment:
Move this test to the airflow module?

Reply:
it uses private method defined the test file. I understand the value of moving the test to airflow module but I've some concern on duplicating code. What's your stance?